.github/workflows/java_binding_publish.yml

```diff
@@ -0,0 +1,140 @@
+name: Create Java release
+
+on:
+  workflow_dispatch:
+    paths:
```

Comment:
The paths thing is not needed with workflow_dispatch?

Reply:
The docs didn't mention it, so let's remove?

Reply:
removed